src/v2/constants.js

```diff
@@ -1,4 +1,6 @@
+import * as solanaWeb3 from '@solana/web3.js';
+
 export const MIN_TERM_LEN = 3;
-export const LAMPORT_SOL_RATIO = 0.0000000000582;
+export const LAMPORT_SOL_RATIO = 1.0 / solanaWeb3.SOL_LAMPORTS;
```

Comment:
There is also a `LAMPORT_SOL_RATIO` in api/util.js
Can you please consolidate into one constant? api is probably the right level for it.
😍  pulling in solanaWeb3.SOL_LAMPORTS, though!